packages/website/ts/blockchain.ts

```diff
@@ -329,7 +330,7 @@ export class Blockchain {
     public async getUnavailableTakerAmountAsync(orderHash: string): Promise<BigNumber> {
         utils.assert(orderHashUtils.isValidOrderHash(orderHash), 'Must be valid orderHash');
         utils.assert(this._contractWrappers !== undefined, 'ContractWrappers must be instantiated.');
-        const unavailableTakerAmount = await this._contractWrappers.exchange.getFilledTakerAssetAmountAsync(orderHash);
+        const unavailableTakerAmount = await this._contractWrappers.exchange.filled.callAsync(orderHash, {});
```

Comment:
Why do we need to pass an empty anonymous object as the second parameter?

Reply:
it's the txData - we can set it to default to an empty object!